nilearn/signal.py

```diff
@@ -473,7 +473,16 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
     # Remove confounds
     if confounds is not None:
         confounds = _ensure_float(confounds)
-        confounds = _standardize(confounds, normalize=True, detrend=detrend)
+        confounds = _standardize(confounds, normalize=standardize,
+                                 detrend=detrend)
+        if not standardize:
+            # Improve numerical stability by controlling the range of
+            # confounds. We don't rely on _standardize as it removes any
+            # constant contribution to confounds
+            confound_max =np.maximum(confounds.max(axis=0),
```

Comment:
Shameful plug, this was caught by the allowed failures build in Travis:
https://travis-ci.org/nilearn/nilearn/jobs/104836962

You can run ./continuous_integration/flake8_diff.sh if you want to make sure you don't introduce errors that flake8 would detect.